tests/unit/ChangeBusinessType.spec.ts

```diff
@@ -46,25 +47,25 @@ describe('Change Business Type component', () => {
 
   it('Should have tooltip for Cooperative', () => {
     // init entity type
-    store.state.stateModel.tombstone.entityType = 'CP'
+    store.stateModel.tombstone.entityType = CorpTypeCd.COOP
```

Comment:
I think I remember Severin you say something about being cautious about using `CorpTypeCd` enums.. 

I did in a couple of spots put in some asserts to check the enum values.

Reply:
I don't remember why I said that, but I think you shouldn't use enums in the unit tests.

Reply:
I disagree with you, but I'll take them out if you'd like

Reply:
Curious what you think about this.

Reply:
I think they should be left in, otherwise the store type definition should be changed. 

Here's a prime example of where ENUM's went wrong recently with AUTH-WEB (not in the unit tests.. but same idea):

https://github.com/bcgov/sbc-auth/pull/2303/files

Reply:
This problem could have been identified at compile time, if we used enums instead of strings there. Or would have been a non issue because the enum value would have been correct versus the string that was entered.

Reply:
Previously, the unit tests would let you get away with untyped objects, etc, so I didn't want to mix types and non-types (in order to make if fragile).

I agree that full typing makes it more fragile, so let's go that route. Change what you're willing to change (thanks in advance) but I realize this is OOS for this PR.